Resin/Analyzer.cs

```diff
@@ -2,18 +2,29 @@
 
 namespace Resin
 {
-    public class Analyzer
+    public class Analyzer:IAnalyzer
     {
+        public static readonly char[] DefaultTakoneSeparators =
+        {
+            ' ', '.', ',', ';', ':', '!', '"', '&', '?', '#', '*', '+', '|', '=', '-', '_', '@', '\'',
+            '<', '>', '�', '�', '�', '`', '(', ')', '[', ']', '{', '}', '/', '\\',
+            '\r', '\n', '\t'
+        };
+
         private readonly char[] _tokenSeparators;
 
+        public Analyzer() : this(DefaultTakoneSeparators)
+        {
+
+        }
+
         public Analyzer(char[] tokenSeparators = null)
```

Comment:
since we have a default ctor and since you don't allow tokenSeparators to be null, it shouldn't be marked as optional. Did the tests show green after this change?